.github/workflows/unifi-os.yml

```
@@ -36,48 +36,28 @@ jobs:
-            yq -I=0 '
-              [.table[].model] | unique |
-                map(select(
-                  strenv(SKU) == "all" or
-                  strenv(SKU) == "" or
-                  . == strenv(SKU)
-                ))
-            ' docker/unifi-os/firmware.json
-      - id: pairs
-        uses: mikefarah/yq@c14f446382944492701b16c1ddb48bb9dbe683e3 # v4
+      - id: matrix
+        uses: $/.github/actions/unifi-os-matrix
```

actionlint (1.7.12)

[error] specifying action "$/.github/actions/unifi-os-matrix" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)